PlexRequests.Services/Interfaces/INotification.cs

```diff
@@ -35,5 +36,7 @@ public interface INotification
         string NotificationName { get; }
 
         Task NotifyAsync(NotificationModel model);
+
+        Task NotifyAsync(NotificationModel model, Settings settings);

```

Comment:
I still don't understand why we need to modify the interface for the settings?

Why can't we just use the `ISettingsService<T>` in the constructor (DI) and then call `GetSettings()`?